docs/BR.md

```diff
@@ -2887,7 +2907,20 @@ This section contains several fields that are common among multiple certificate
 
 ##### 7.1.2.11.2 CRL Distribution Points
 
-If present, the CRL Distribution Points extension MUST contain at least one `DistributionPoint`; containing more than one is NOT RECOMMENDED. All `DistributionPoint` items must be formatted as follows:
+The CRL Distribution Points extension MUST be present in:
+- Subordinate CA Certificates; and
+- Subscriber Certificates that 1) do not qualify as "Short-lived Subscriber Certificates" and 2) do not include an Authority Information Access extension with an id-ad-ocsp accessMethod.
+
+The CRL Distribution Points extension SHOULD NOT be present in:
+- Root CA Certificates.
+
+The CRL Distribution Points extension is OPTIONAL in:
+- Short-lived Subscriber Certificates.
+
+The CRL Distribution Points extension MUST NOT be present in:
+- OCSP Responder Certificates.
+
+When present, the CRL Distribution Points extension MUST contain at least one `DistributionPoint`; containing more than one is NOT RECOMMENDED. All `DistributionPoint` items must be formatted as follows:
```

Comment:
```suggestion
When present, the CRL Distribution Points extension MUST contain at least one `DistributionPoint`; containing more than one is NOT RECOMMENDED. All `DistributionPoint` items MUST be formatted as follows:
```
maybe? I'm not sure why this "must" wasn't capitalized previously.

Reply:
It was a typo on my part dating back to the [early versions](https://github.com/ryancdickson/staging/blob/36bd26adf1346fcb9ff23e43468501778a7d3d7c/docs/BR.md#:~:text=When%20present%2C%20the%20CRL%20Distribution%20Points%20extension%20MUST%20contain%20at%20least%20one%20DistributionPoint%3B%20containing%20more%20than%20one%20is%20NOT%20RECOMMENDED.%20All%20DistributionPoint%20items%20must%20be%20formatted%20as%20follows%3A) of the proposal.